Docs for simulating multiple chromosomes with pedigrees

apragsdale opened the pull request:
This is very much a work in progress, but wanted to get something sketched out. 

Closes #2082

jeromekelleher commented on docs/ancestry.md:
```diff
@@ -688,9 +688,58 @@ purposes.
+This also results in simpler, reusable code which can be parallelized. Finally,
+this approach also improves efficiency over a single simulation of multiple
+concatenated chromosomes, as computational costs scale as the square of the
+sequence length (to check: right?).
```
Yep, that's right.

apragsdale replied:
Thanks!

jeromekelleher reviewed (commented):
Very nice, thanks. Hadn't appreciated that using the pedigree captured the between chrom correlations :+1:

apragsdale commented:
> Very nice, thanks. Hadn't appreciated that using the pedigree captured the between chrom correlations +1

Yes! I think the fixed pedigree model really shines in this type of application. It's a really versatile tool.

I think this is ready for a more thorough review. Happy to make any changes to the larger structure of how I've laid out this section as well if you think there is a more natural order of presenting things.

jeromekelleher reviewed (approved):
LGTM!

I guess we should probably provide an example of how to do that NaN-bits extraction, but we can log that as a follow-up issue?

apragsdale commented:
Great! Let's open a follow-up issue for docs on handling the NaN bits, and I can work on that when I've got a bit of time in the coming weeks.

I just rebased and squashed this one.
